crates/viewer/re_viewer/src/ui/settings_screen.rs

```diff
@@ -160,6 +138,102 @@ fn settings_screen_ui_impl(ui: &mut egui::Ui, app_options: &mut AppOptions, keep
     }
 }
 
+fn video_section_ui(ui: &mut Ui, app_options: &mut AppOptions) {
+    #[cfg(not(target_arch = "wasm32"))]
+    {
+        ui.re_checkbox(
+            &mut app_options.video_decoder_override_ffmpeg_path,
+            "Override the FFmpeg binary path",
+        )
+        .on_hover_ui(|ui| {
+            ui.markdown_ui(
+                "By default, the viewer tries to automatically find a suitable FFmpeg binary in \
+                the system's `PATH`. Enabling this option allows you to specify a custom path to \
+                the FFmpeg binary.",
+            );
+        });
+
+        ui.add_enabled_ui(app_options.video_decoder_override_ffmpeg_path, |ui| {
+            ui.horizontal(|ui| {
+                // TODO(ab): needed for alignment, we should use egui flex instead
+                ui.set_height(19.0);
+
+                ui.label("Path:");
+
+                ui.add(egui::TextEdit::singleline(
+                    &mut app_options.video_decoder_ffmpeg_path,
+                ));
+            });
+        });
+
+        ffmpeg_path_status_ui(ui, app_options);
+    }
+
+    // This affects only the web target, so we don't need to show it on native.
+    #[cfg(target_arch = "wasm32")]
+    {
+        let hardware_acceleration = &mut app_options.video_decoder_hw_acceleration;
+        ui.horizontal(|ui| {
+            ui.label("Decoder:");
+            egui::ComboBox::from_id_salt("video_decoder_hw_acceleration")
+                .selected_text(hardware_acceleration.to_string())
+                .show_ui(ui, |ui| {
+                    ui.selectable_value(
+                        hardware_acceleration,
+                        re_video::decode::DecodeHardwareAcceleration::Auto,
+                        re_video::decode::DecodeHardwareAcceleration::Auto.to_string(),
+                    ) | ui.selectable_value(
+                        hardware_acceleration,
+                        re_video::decode::DecodeHardwareAcceleration::PreferSoftware,
+                        re_video::decode::DecodeHardwareAcceleration::PreferSoftware.to_string(),
+                    ) | ui.selectable_value(
+                        hardware_acceleration,
+                        re_video::decode::DecodeHardwareAcceleration::PreferHardware,
+                        re_video::decode::DecodeHardwareAcceleration::PreferHardware.to_string(),
```

Comment:
`use re_video::decode::DecodeHardwareAcceleration;` at the top of this file would really help readability